todo_app/app.py

```diff
@@ -20,8 +69,68 @@ def add_item_page():
 @app.route('/addItem', methods=['POST'])
 def add_item():
     item = request.form.get('item') 
-    session.add_item(item)
+    #Add item to list to do
+    add_item_url = "https://api.trello.com/1/cards"
+    add_item_query = trello_key_params
+    add_item_query['idList'] = trello_todo_list_id
+    add_item_query['name'] = item
+    add_item_resp = query_trello("POST", add_item_url, trello_key_params)
+    return redirect(url_for('index'))
+
+@app.route('/createTodoList', methods=['GET'])
+def createTodoList():
+    new_list_url = "https://api.trello.com/1/lists"
+    new_list_query_params = trello_key_params
+    new_list_query_params['name'] = "To Do"
+    new_list_query_params['idBoard'] = trello_board_id
+    new_list_resp = query_trello("POST", new_list_url, new_list_query_params)
+    if 200 != new_list_resp.status_code:
+        return render_template('error.html')
+    global trello_todo_list_id
+    resp_dict = json.loads(new_list_resp.text)
+    trello_todo_list_id = resp_dict['id']
+    return redirect(url_for('index'))
+
+@app.route('/complete_item/<id>', methods=['GET'])
+def complete_item(id):
+    #print(id)
+    #We have the ID of the card
+    #Now we want to add it to the completed list and remove it from the To Do list
+
+    global trello_done_list_id
```

Comment:
If your function tries to assign to the variable, then Python assumes you want a local variable with the same name (hiding the global one). I think that's the correct default behaviour because you rarely want a function to access global variables. You're more likely reusing a common variable name.

If you are just reading the value, then you don't need the `global trello_done_list_id`

What you've done is a decent solution to the problem, but to make it as neat as possible you might move all of the Trello interaction into a `TrelloClient` class and then the list ids would be class attributes, not global variables.